src/hooks/useGovernanceProfile.ts

```diff
@@ -19,7 +24,7 @@ export default function useGovernanceProfile(profileAddress?: string | null) {
         return await Governance.get().getUserProfile(profileAddress)
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
       } catch (error: any) {
-        if (error.statusCode !== 404) {
+        if (!isNotFoundOrBadRequest(error)) {
```

Comment:
are we sure that error is always of type `RequestError`?

Reply:
It should do so now, with the changes in the back